site/app/models/gradeable/Gradeable.php

```diff
@@ -96,6 +96,16 @@
  * @method int getLimitedAccessBlind()
  * @method void setPeerBlind($peer_blind)
  * @method int getPeerBlind()
+ * @method void setPeerAutograding($peer_autograding)
+ * @method boolean getPeerAutograding()
```

Comment:
```suggestion
 * @method bool getPeerAutograding()
```

Apply for others

Reply:
Alright, updated (but it was passing all checks before, this might not be the reason for failing tests)